lib/src/theme/mix/mix_tokens.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:meta/meta.dart';
+
+import '../tokens/color_token.dart';
+import '../tokens/radius_token.dart';
+import '../tokens/space_token.dart';
+import '../tokens/text_style_token.dart';
+
+@immutable
+class MixTokens {
+  final color = const _MixColorTokens();
+  final textStyle = const _MixTextStyles();
+  final space = const _MixSpaces();
+  final radius = const _MixRadii();
+
+  const MixTokens();
+}
+
+@immutable
+class _MixColorTokens {
+  final primary = const ColorToken('mix.color.primary');
```

Comment:
These would not be Mix Tokens but Material ones. I don't think we should use them as MixTokens by default but maintain the ability to import material tokens into it.

Reply:
done